src/applications/login/app-entry.jsx

```diff
@@ -0,0 +1,14 @@
+import '../../platform/polyfills';
+import './sass/login-page.scss';
+
+import startApp from '../../platform/startup';
+
+import routes from './routes';
+import reducer from './reducers';
+import manifest from './manifest.json';
+
+startApp({
+  url: manifest.rootUrl,
+  reducer,
```

Comment:
I know this is part of the larger framework of `startApp` and the app manifest files, but I'm wondering if there's any option to remove this entirely? I don't think this page is going to necessarily need Redux, and removing it would just reduce some unnecessary boilerplate

Reply:
I don't know if/what side effect there could be, but you could render to the DOM using React yourself using `React.renderToDOM('#react-root,...)` and not use `startApp`. That's what we use to do before the code was platform-ized.

Reply:
You can find one of the more unconventional `*-entry` files in `applications/static-pages`, which doesn't use `startApp` and is really just a collection of smaller React apps. They might individually use their own stores, but you'll see that there's not really anything forcing them to use Redux.